argo-cd-apps/overlays/stage-ext/kustomization.yaml

```
@@ -0,0 +1,17 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
```

Kustomize build is failing, should be:
```suggestion
  - ../base
```